cookbooks/fb_cron/recipes/default.rb

```diff
@@ -26,7 +26,10 @@
 when 'mac_os_x'
   svc_name = 'com.vix.cron'
 when 'rhel'
-  package_name = node['platform_version'] >= 6 ? 'cronie' : 'vixie-cron'
+  package_name = (
```

Comment:
once a ternary operator spans more than one line, it's time to just make an `if` statement. :)